ci/validate_schema.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Run from top level dir
+cd "$(dirname $0)/.."
+
+JSON=$(cat ./conjur-oss/values.schema.json)
+
+if jq -e . >/dev/null 2>&1 <<<"${JSON}"; then
+    echo "JSON Schema is valid." && exit 0
+else
+    echo "Failed to parse JSON, or got false/null" && exit 1
+fi
```

Comment:
```suggestion
if jq -e . >/dev/null < ./conjur-oss/values.schema.json; then
    echo "Helm chart values schema is valid JSON." && exit 0
else
    echo "Helm chart values schema is not valid JSON." && exit 1
fi
```

Reply:
Some suggestions
1. Condense jq invocation
2. To be more descriptive with the messages